src/cc-compat.h

```diff
@@ -48,6 +48,11 @@
 /* Define the printf format specifer to use for size_t output */
 #if defined(_MSC_VER) || defined(__MINGW32__)
 
+/* Visual Studio 2012 and prior lack PRId64 entirely */
+#	if (_MSC_VER < 1800)
```

Comment:
Wouldn't it be easier to just check if `PRId64` is defined instead of relying on the `_MSC_VER`? No idea though if MSVC is using e.g. a typedef, but I guess you'd know :)

Reply:
How would you implement `PRId64` using a _typedef_ (in C)? AFAIK the only way to implement it is by using preprocessor-macros.

Reply:
How would you implement `PRId64` using a _typedef_ (in C)? AFAIK the only way to implement it is by using preprocessor-macros.